### ISSUE TITLE
Bug: Combobox does not work properly with VoiceOver

### ISSUE THREAD

**Issue by Jonas-C** [chakra-ui/zag#1664]
# 🐛 Bug report

It's more or less impossible to select a combobox value with a screen reader. The combobox does not open when pressing ctrl+option+space. Navigating list arrows with ctrl+option+arrow works, but navigation with only arrows does not. Selecting the highlighted item with ctrl+option+space does nothing. 

## 💻 Link to reproduction

https://zagjs.com/components/react/combobox

## 🧐 Expected behavior

I'd like to navigate the combobox with a screen reader!

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Zag Version      | 0.61.0            |
| Browser          | Firefox/Safari           |
| Operating System | MacOS            |

## 📝 Additional information

See https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-none/

**Comment by segunadebayo** [comment]
I just pushed a fix for this. We'll release an update shortly.

If the issue persists after upgrading, I'll re-open it.
